### ISSUE TITLE
Too strict validation of image/tag when updating containers

### ISSUE THREAD

**Issue by kwando** [scaleway/scaleway-cli#2068]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

## Command attempted

```
$ scw container container update 573f2ca8-643f-4759-8c9f-310381013 registry-image=rg.fr-par.scw.cloud/hello:0.0.0 region=fr-par
Registry images should be rg.(fr-par or nl-ams).scw.cloud/<alphanumeric - _>/<alphanumeric - _>:<alphanumeric - _ .>
```

### Expected Behavior

I expect the container deployment to be updated with the new version.

### Actual Behavior

I get an error message complaining about alphanumeric characters (my image has the tag `0.0.1`)

## More info
The image with that tag can be deployed through the UI/dashboard so I think this is CLI issue.
```
scw version
Version    2.4.0
BuildDate  unknown
GoVersion  go1.17.2
GitBranch  unknown
GitCommit  unknown
GoArch     arm64
GoOS       darwin
```

**Comment by remyleone** [comment]
Hello,
Could you try with:
`$ scw container container update 8c990e63-c9a9-428a-b808-dfa9fe5508a3 registry-image="rg.fr-par.scw.cloud/IDofNamespace/hello:0.0.0" region=fr-par -D`

I think your endpoint might be missing for instance a registry endpoint for me is `rg.fr-par.scw.cloud/funcscwclicnsferventbanachfx2z76qi`
Within this registry you can look for the hello image id with: `rg.fr-par.scw.cloud/funcscwclicnsferventbanachfx2z76qi/hello:0.0.0`

**Comment by kwando** [comment]
Right! The `namespace` got accidentally dropped in my image names. If I add it it works as expected.

Thanks for your help and sorry for the noise.
